packages/core/src/rules/oas3/__tests__/no-invalid-media-type-examples.test.ts

```diff
@@ -458,6 +460,8 @@ describe('no-invalid-media-type-examples', () => {
                       examples:
                         first:
                           value: {}
+                        second:
```

Comment:
Could you also create a separate test for the case instead of extending the existing ones? I'm afraid that the descriptions might not be too helpful. 
Please don't forget to add the case when both `externalValue` and `value` (correct and incorrect) are defined.

Reply:
Done in 31758ac4a22e214c6667ba41717cffe9867b917b.